lib/confluence/aggressive_removal_data.es6.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  name: 'AggressiveRemovalData',
+  package: 'org.chromium.apis.web',
+  documentation: `This class contains a browser and its
+    number of failure to ship APIs.`,
+  ids: ['browserName', 'date'],
+  properties: [
+    {
+      name: 'browserName',
+      class: 'String',
+      documentation: 'Name of the Browser',
+      required: true,
+      final: true,
+    },
+    {
+      name: 'browserOneYearAgo',
```

Comment:
I think this should have `class: 'FObjectProperty'`, `of: 'path.to.BrowserObjectType'`. @kgrgreer is that the right `class`, or should it be `FObject`?